Sources/Aardvark/BugReporting/LogStoreAttachmentGenerator.swift

```diff
@@ -107,11 +110,13 @@ public final class LogStoreAttachmentGenerator: NSObject {
     /// - parameter logMessages: The log messages to be included in the attachment.
     /// - parameter logFormatter: The formatter with which to format the log messages.
     /// - parameter logStoreName: The name of the log store from which the logs were collected.
-    @objc(attachmentForLogMessages:usingLogFormatter:logStoreName:)
+    /// - parameter numberOfErrorsInHighlights: The number of errors to include in the highlights for the log store.
+    @objc(attachmentForLogMessages:usingLogFormatter:logStoreName:numberOfErrorsInHighlights:)
```

Comment:
This is a breaking change for Aardvark. This one would be fairly easy to work around (by creating a separate method, rather than adding a parameter with a default value). I could go either way on this one.

Reply:
2¢ For such a small change, I think the workaround is preferable.